docs/snippets/ov_common.py

```diff
@@ -105,5 +108,6 @@ def callback(request, frame_id):
 #! [ov_api_2_0:get_output_tensor_v10]
 
 #! [ov_api_2_0:load_old_extension]
-core.add_extension("path_to_extension_library.so")
+path_to_extension_library = get_path_to_extension_library()
```

Comment:
@jiwaszki should the line be out of snippet code? like you suggested for `model = get_model()`

Reply:
I think yes, will move it out of the snippet code